Ryujinx.Ava/Input/AvaloniaKeyboardMappingHelper.cs

```diff
@@ -167,15 +167,13 @@ static AvaloniaMappingHelper()
 
         public static bool TryGetAvaKey(Key key, out AvaKey avaKey)
         {
-            var keyExist = (int)key < _keyMapping.Length;
+            avaKey = AvaKey.None;
+
+            bool keyExist = (int)key < _keyMapping.Length;
```

Comment:
```suggestion
            bool keyExist = (int)key < _keyMapping.Length;
            avaKey = AvaKey.None;

```
I think @AcK77 would prefer this formatting. :p

Reply:
Since `avaKey` is an out param I'd prefer to set it at the beginning